install/kubernetes/cilium/values.yaml.tmpl

```
@@ -3816,6 +3816,9 @@ clustermesh:
       # -- base64 encoded PEM values for the clustermesh-apiserver server certificate and private key.
       # Used if 'auto' is not enabled.
       server:
+        # -- Name of the Secret containing the clustermesh-apiserver server certificate and private key.
+        # If specified, cert and key are ignored.
+        existingSecret: ""
```

I'd suggest adding a validation to fail loudly if both `auto.enabled` and `existingSecret` are set, to prevent surprises. Especially because the existing `Used if 'auto' is not enabled.` comment would not be true anymore.